be/src/formats/parquet/level_builder.cpp

```diff
@@ -357,8 +357,9 @@ Status LevelBuilder::_write_datetime_column_chunk(const LevelBuilderContext& ctx
     DeferOp defer([&] { delete[] values; });
 
     for (size_t i = 0; i < col->size(); i++) {
-        // normalize to utc
-        auto timestamp = timestamp::sub<TimeUnit::SECOND>(data_col[i]._timestamp, _offset);
+        auto timestamp = use_int96_timestamp_encoding
```

Comment:
I don't understand this. Could you elaborate it a little bit?

Reply:
The   use_int96_timestamp_encoding   indicates the use of   int96   to represent the   timestamp   type. This is a deprecated encoding method, but we need to maintain compatibility. For timestamps encoded in this way, we continue to use the previous logic and convert the data to the UTC time zone when writing.

Reply:
Why it needs to be converted the data to the UTC timezone when it's encoding in int96